home.html

```diff
@@ -0,0 +1,327 @@
+            </div>
+          </section>
+        </section>
+      </section>
+      <section class="right">
+        <div class="top-selling-card">
+          <div class="card-img">
+            <img src="elements/girl.jpg" alt="Rome,Italy Image" />
+          </div>
+          <div class="card-texts">
+            <h3>Trip To Greece</h3>
+            <div class="destination-details">
+              <p>14-29 June| by Robbin jo</p>
+            </div>
+            <div class="icons">
+              <img src="elements/LEAF.png" alt="send-icon" />
+              <img src="elements/map.png" alt="" style="width: 25%" />
+              <img src="elements/send.png" alt="send-icon" />
+            </div>
+            <div class="trip-days">
+              <img src="elements\building.png" />
+              <p>24 people going</p>
+              <img
+                src="elements/heart (6) 1.png"
+                alt="like"
+                style="margin-left: 34%; width: 5%"
+              />
+            </div>
+          </div>
+        </div>
+        <section class="overhead-banner">
+          <section class="card">
+            <section class="aside-image">
+              <img src="elements/rome-sm.png" alt="rome" />
+            </section>
+            <section class="aside-text">
+              <h6 style="margin-top: 20px">Ongoing</h6>
+              <h4 style="margin: 0%">Trip to rome</h4>
+              <h6 style="margin-top: 8px; display: inline; color: #b4a7f8">
+                40%
+              </h6>
+              <h6 style="display: inline; color: black">completed</h6>
+              <div class="progress-line">
+                <div class="completed-bar"></div>
+              </div>
+            </section>
+          </section>
+        </section>
+      </section>
+    </section>
+    <div class="sub-container">
+      <div class="telegram"><img src="elements/Group 77.png" alt="tele" /></div>
+      <section class="subscribe">
+        <div class="rings"><img src="elements/Group 42.png" alt="rings" /></div>
+        <div class="right-r">
+          <img src="elements/Group 42.png" alt="rings" />
+        </div>
+        <section class="heading">
+          Subscribe to get information, latest news and other <br />
+          interesting offers about Cobham
+        </section>
+        <section class="mailer">
+          <div class="email-typer">
+            <img src="elements/email.png" alt="" /><input
+              type="email"
+              placeholder="Your email"
+            />
+          </div>
+          <div class="Subscribe-btn">Subscribe</div>
+        </section>
+      </section>
+    </div>
+    <footer class="footer">
+      <div class="f1">
+        <h1>TravelPro</h1>
+        <p>
+          Book your trip in a minute,get full <br />
+          control for much longer.
+        </p>
+      </div>
+      <div class="about">
+        <h3>Company</h3>
+        <h5><a href="www.google.com">About</a></h5>

```

Comment:
```suggestion
        <h5><a href="#">About</a></h5>

```

Check why we need to keep # if we don't know the route and make these changes for below as well

Reply:
resolved